internal/cmd/billing/command_cost_list.go

```diff
@@ -0,0 +1,129 @@
+package billing
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/cli/internal/pkg/billing"
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+type costOut struct {
+	StartDate           string `human:"Start Date" serialized:"start_date"`
+	EndDate             string `human:"End Date" serialized:"end_date"`
+	Granularity         string `human:"Granularity" serialized:"granularity"`
+	LineType            string `human:"Line Type" serialized:"line_type"`
+	Product             string `human:"Product,omitempty" serialized:"product,omitempty"`
+	ResourceId          string `human:"Resource ID,omitempty" serialized:"resource_id,omitempty"`
+	ResourceDisplayName string `human:"Resource Display Name,omitempty" serialized:"resource_display_name,omitempty"`
+	EnvironmentId       string `human:"Environment ID,omitempty" serialized:"environment_id,omitempty"`
+	NetworkAccessType   string `human:"Network Access Type,omitempty" serialized:"network_access_type,omitempty"`
+	Price               string `human:"Price,omitempty" serialized:"price,omitempty"`
+	OriginalAmount      string `human:"Original Amount" serialized:"original_amount"`
+	DiscountAmount      string `human:"Discount Amount,omitempty" serialized:"discount_amount,omitempty"`
+	Amount              string `human:"Amount,omitempty" serialized:"amount,omitempty"`
+}
+
+func (c *command) newCostListCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "list --start-date <start-date> --end-date <end-date>",
+		//Example: "list  --start-date 2023-01-01  --end-date 2023-01-10",
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `List Billing costs from 2023-01-01 to 2023-01-10:`,
+				Code: "confluent billing list --start-date 2023-01-01 --end-date 2023-01-10",
+			}),
```

Comment:
Can you move this to the bottom of the struct and follow the same format as other commands?